scripts/ci/upload-build

```diff
@@ -2,35 +2,33 @@
 set -ex
 
 # configuration
-BUILD_VARIANT=${BUILD_VARIANT:-''}
-BRANCH_NAME=${BRANCH_NAME:-'master'}
-DIST_PATH=${DIST_PATH:-'dist'}
-PACKAGE_PATH=${PACKAGE_PATH:-'.'}
-SHA_SLUG=${SHA_SLUG:-$(git rev-parse --short HEAD)}
-
-# aws settings
 AWS_BUCKET='downloads.mesosphere.io'
 BUCKET_PATH='dcos-ui'
+BRANCH_NAME=${BRANCH_NAME:-'master'}
+PKG_VERSION=$(git tag -l --contains HEAD)
+
+if [[ $PKG_VERSION ]]
+then
+  PKG_VERSION="-$PKG_VERSION"
+fi
 
-# build release filename
-FILENAME='dcos-ui'
-PKG_VERSION=$(python -c "import sys, json; \
-  pkg = json.load(open('$PACKAGE_PATH/package.json')); \
-  sys.stdout.write(str(pkg['version']));")
 PKG_BRANCH=$(echo ${BRANCH_NAME} | sed -E 's/([^/]*)$|./\1/g')
-RELEASE_NAME="${PKG_BRANCH}+${FILENAME}${BUILD_VARIANT}-v${PKG_VERSION}+${SHA_SLUG}.tar.gz"
+
+# ex. master+dcos-ui-v1.12.0-rc.11.tar.gz
+# ex. master+dcos-ui-ee-v1.12.0-rc.11.tar.gz
+RELEASE_NAME="${PKG_BRANCH}+dcos-ui${PKG_VERSION}.tar.gz"
```

Comment:
if this script is called with a tag (e.g. `BRANCH_NAME=v1.12.0-rc.1`) this will result in a broken filename (e.g. `v1.12.0-rc.1+dcos-ui-v1.12.0-rc.1.tar.gz` instead of `master+dcos-ui-v1.12.0-rc.1.tar.gz`) :(

Reply:
The Idea behind these filenames was to be able to have different version numbers for different branches. Imagine we use semver and there is a DC/OS UI version `1.5.3` which is released for DC/OS `1.11`, so the release name will be `1.11+dcos-ui-v1.5.3-somesha.tar.gz`.

Reply:
_(hit shift+enter again…)_

now imagine that we branched 1.11 on version `1.5.3` and on `master` (1.12), we're already on version `1.5.8`, but we want to cut a new release for DC/OS 1.11, the next `semver` release would be: `1.11+dcos-ui-1.5.4-somesha.tar.gz`

now, we have two `1.5.4` releases, one for DC/OS 1.11 and a second one for DC/OS "Next" (1.12), both files have to have a different filename, which was archivieved by the old script. the bucket would contain the following releases:
`1.11+dcos-ui-v1.5.3-shomesha.tar.gz`
`1.11+dcos-ui-v1.5.4-shomesha.tar.gz`
`master+dcos-ui-v1.5.4-shomesha.tar.gz`
`…`
`master+dcos-ui-v1.5.8-shomesha.tar.gz`
notice to "duplicate" on 1.5.4.

we are aware that this isnt a goot solution, but its the situation we have to deal with and for us it was the best compromise dealing with the issue. we basicly tried divide our releases by DC/OS target branch and handle them independently from each other. 

And, at last, sorry for not communitcatin this to you guys, this was discussed between orlando and myself and we (I) simply forgot to tell you guys why we went this way - I assumed you guys dont touch this script because it wasnt part of "your" work :( Apologies!

Reply:
To move forward, we could simply remove the `PKG_BRANCH` part from the filename, since we are currently using the old version scheme filenames like `dcos-ui-v1.12.0-rc.1` should be okay…

Reply:
@juliangieseke I thought we wanted to do this refactoring in one of the next tickets next sprint

Reply:
yes, but I wasnt asking to refactor everything right now?!
The filenames are broken and this has to be fixed. Just wanted to provide some background info :)

Reply:
Cleaned up everything and added short notice that it is background info & what i think would be a proper fix right now. 👍

Reply:
Ah okay, thank you for explaining, I got you now 👍

Reply:
lets get this merged anf fix the naming stuff later, I've created a ticket in pulled it into the sprint 👍 https://jira.mesosphere.com/browse/DCOS-21766